book/http.md

```diff
@@ -142,7 +141,7 @@ Requesting information
 Once it's connected, the browser requests information from the server
 by giving its *path*, the path being the part of a URL that comes
 after the host name, like `/index.html`. The structure of the request
-is shown below. You can type it into `telnet` (make sure to type a blank line after the `Host` line):
+is shown in Figure 2. You can type it into `telnet` to try it.
```

Comment:
Copyeditor said "you can type this" instead of "you can type it" but I think "it" makes more sense.